docs/extend/plugins_authorization.md

```diff
@@ -58,7 +58,7 @@ respectively.
 
 ## Default user authorization mechanism
 
-If TLS is enabled in the [Docker daemon](../security/https.md), the default user authorization flow extracts the user details from the certificate subject name.
+If TLS is enabled in the [Docker daemon](https://docs.docker.com/engine/security/https.md), the default user authorization flow extracts the user details from the certificate subject name.
```

Comment:
I don't think it will quite work like this. Change the URLs to https://docs.docker.com/engine/security/https/ and so on. Where you see a `.md`, change it to just a `/` instead. Make sense? All the URLs below should be changed to the same pattern. You can test the URL if in doubt.